bin/travis-build.bash

```diff
@@ -41,7 +48,12 @@ sudo -u postgres psql -c 'CREATE DATABASE ckan_test WITH OWNER ckan_default;'
 
 echo "Initialising the database..."
 cd ckan
-paster db init -c test-core.ini
+if [ $CKANVERSION == 'master' ]
+then
+    ckan -c test-core.ini db init
```

Comment:
should be checking > '2.8' instead

Reply:
Good point. I'll use `2.9` instead, as in 2.8 we were mainly using paster. For the consistency, I'll leave single bracket there, but it will result in a bit ugly condition block:
`if [ $CKANVERSION \< '2.9' ]...`
In this case, `<` must be escaped, otherwise, it will work as input redirect. double-brackets would solve the problem, but we are using single-brackets in all the extensions and I'm not so confident in my sh knowledge to guarantee, that switching into a bit different command won't create unexpected side-effects.